feature-libs/checkout/core/store/actions/checkout.action.ts

```diff
@@ -33,12 +33,27 @@ export const CLEAR_CHECKOUT_DELIVERY_MODE_SUCCESS =
 export const CLEAR_CHECKOUT_DELIVERY_MODE_FAIL =
   '[Checkout] Clear Checkout Delivery Mode Fail';
 
+/**
+ * @deprecated since 4.3.0. Add delivery address is handled with command in CheckoutDeliveryService.
```

Comment:
Is 4.3 even planned for? 🤔

Reply:
We will update that comments when we will know when we want to merge it :)

I assume we will have 3 minor before major